qa/L0_backend_python/test.sh

```diff
@@ -447,85 +278,35 @@ $page_size."
         RET=1
     fi
 done
-set +e
 
 kill $SERVER_PID
 wait $SERVER_PID
-sleep 5
 
-rm -fr ./models
-rm -rf *.tar.gz
-apt update && apt install software-properties-common rapidjson-dev -y
-wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | \
-	gpg --dearmor - |  \
-	tee /etc/apt/trusted.gpg.d/kitware.gpg >/dev/null && \
-	apt-add-repository 'deb https://apt.kitware.com/ubuntu/ focal main' && \
-	apt-get update && \
-	apt-get install -y --no-install-recommends \
-	cmake-data=3.18.4-0kitware1ubuntu20.04.1 cmake=3.18.4-0kitware1ubuntu20.04.1
-install_conda
-
-# Create a model with python 3.9 version
-create_conda_env "3.9" "python-3-9"
-conda install numpy=1.20.1 -y
-create_python_backend_stub
-conda-pack -o python3.9.tar.gz
-path_to_conda_pack=`pwd`/python3.9.tar.gz
-mkdir -p models/python_3_9/1/
-cp ../python_models/python_version/config.pbtxt ./models/python_3_9
-(cd models/python_3_9 && \
-          sed -i "s/^name:.*/name: \"python_3_9\"/" config.pbtxt && \
-          echo "parameters: {key: \"EXECUTION_ENV_PATH\", value: {string_value: \"$path_to_conda_pack\"}}">> config.pbtxt)
-cp ../python_models/python_version/model.py ./models/python_3_9/1/
-cp python_backend/builddir/triton_python_backend_stub ./models/python_3_9
-conda deactivate
-
-# Create a model with python 3.6 version
-create_conda_env "3.6" "python-3-6"
-conda install numpy=1.18.1 -y
-conda-pack -o python3.6.tar.gz
-path_to_conda_pack=`pwd`/python3.6.tar.gz
-create_python_backend_stub
-mkdir -p models/python_3_6/1/
-cp ../python_models/python_version/config.pbtxt ./models/python_3_6
-(cd models/python_3_6 && \
-          sed -i "s/^name:.*/name: \"python_3_6\"/" config.pbtxt && \
-          echo "parameters: {key: \"EXECUTION_ENV_PATH\", value: {string_value: \"$path_to_conda_pack\"}}" >> config.pbtxt)
-cp ../python_models/python_version/model.py ./models/python_3_6/1/
-cp python_backend/builddir/triton_python_backend_stub ./models/python_3_6
-
-run_server
-if [ "$SERVER_PID" == "0" ]; then
-    echo -e "\n***\n*** Failed to start $SERVER\n***"
-    cat $SERVER_LOG
-    exit 1
+(cd env/ && bash -ex ./test.sh)
```

Comment:
extra '/' after env. Here and in other places as well. Also './' is not needed before test.sh

Reply:
Fixed.